presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPruneIndexSourceColumns.java

```diff
@@ -85,7 +88,9 @@ private static PlanNode buildProjectedIndexSource(PlanBuilder p, Predicate<Symbo
                 p.indexSource(
                         new TableHandle(
                                 new ConnectorId("local"),
-                                new TpchTableHandle("orders", TINY_SCALE_FACTOR)),
+                                new TpchTableHandle("orders", TINY_SCALE_FACTOR),
+                                TestingTransactionHandle.create(),
+                                Optional.of(TestingHandle.INSTANCE)),
```

Comment:
If we want to keep `IndexSourceNode.tableLayout` be consistent with `IndexSourceNode.tableHandle.layout`, we should use `Optional.empty()` here :)